operator-framework/src/test/java/io/javaoperatorsdk/operator/CRDMappingInTestExtensionIT.java

```diff
@@ -43,7 +43,7 @@ void correctlyAppliesManuallySpecifiedCRD() {
   @Group("crd.example")
   @Version("v1")
   @Kind("Test")
-  private static class TestCR extends CustomResource<Void, Void> implements Namespaced {
+  public static class TestCR extends CustomResource<Void, Void> implements Namespaced {
```

Comment:
Is this change necessary? I think we used different visibility on purpose to check if this was possible to do…

Reply:
Found the root cause here https://github.com/fabric8io/kubernetes-client/blob/main/crd-generator/api-v2/src/main/java/io/fabric8/crdv2/generator/CustomResourceInfo.java#L168

`getDeclaredConstructor()` does only work with public classes.